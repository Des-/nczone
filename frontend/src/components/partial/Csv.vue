<template>
  <span><span v-for="(item, idx) in list">{{ item }}<span v-if="idx+1 < list.length">, </span></span></span>
</template>
<script>
export default {
  name: 'nczone-csv',
  props: {
    list: {
      type: Array,
      required: true,
    },
  }
}
</script>
