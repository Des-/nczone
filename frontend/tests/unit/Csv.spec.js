import { shallowMount } from '@vue/test-utils'
import Csv from '@/view/components/Csv'

describe('Csv.vue', () => {
  const list = ['some Data ', 'in a list']
  const cmp = shallowMount(Csv, {
    propsData: { list }
  })

  test('Should match snapshot', () => {
    expect(cmp.html()).toMatchSnapshot()
  })
})
