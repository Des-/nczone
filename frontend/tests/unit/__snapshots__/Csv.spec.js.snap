// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Csv.vue Should match snapshot 1`] = `<span><span>some Data <span>, </span></span><span>in a list<!----></span></span>`;
